parachain/pallets/commitments/src/lib.rs

```diff
@@ -107,15 +110,13 @@ impl<T: Config> Module<T> {
 		let messages: Vec<Message> = <Self as Store>::MessageQueue::take();
```

Comment:
Not too related to this PR, though could be included here - can we skip creating a commitment if the MessageQueue is empty (eg: in the scenario where a few 100 blocks pass and there is no activity)?

Reply:
Fixed :+1: